.pre-commit-config.yaml

```diff
@@ -101,7 +101,7 @@ repos:
       - id: pyupgrade
         args: ["--keep-percent-format"]
   - repo: https://github.com/PyCQA/isort
-    rev: 5.10.1
+    rev: 5.12.0
```

Comment:
Pls rebase, this has been changed directly on `16.0` now ;)

Reply:
okey! thanks :D